localstack/services/ses/provider.py

```diff
@@ -291,3 +293,20 @@ def send_raw_email(
         )
 
         return SendRawEmailResponse(MessageId=message.id)
+
+    @handler("CloneReceiptRuleSet")
+    def clone_receipt_rule_set(
+        self,
+        context: RequestContext,
+        rule_set_name: ReceiptRuleSetName,
+        original_rule_set_name: ReceiptRuleSetName,
+    ) -> CloneReceiptRuleSetResponse:
+        backend = get_ses_backend(context)
+
+        backend.create_receipt_rule_set(rule_set_name)
+        original_rule_set = backend.describe_receipt_rule_set(original_rule_set_name)
+
+        for rule in original_rule_set["Rules"]:
+            backend.create_receipt_rule(rule_set_name, rule)
+
+        return {}
```

Comment:
You must return an instance of `CloneReceiptRuleSetResponse`

Reply:
Hello @viren-nadkarni, hope you are doing well.
Thank you for your comment. I changed the response, but I am not sure what to put inside it.

Also I keep falling into this error where it says there is a duplicate rule set. I changed the names and it still doesn't work. May you please help me fix this?
![image](https://user-images.githubusercontent.com/29195190/194832412-d4175186-f9c1-4588-8e76-9d633a13b1c3.png)

Reply:
The problem might lie in an earlier error message:

```
list indidces must be integers or slices, not str
```

Might be worth a look?

Reply:
Thank you so much @viren-nadkarni !
Your guide was super helpful for figuring the code out 😆 
![image](https://user-images.githubusercontent.com/29195190/194873010-e8529e01-e07d-451b-a0f8-480e3514c8b9.png)